src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/SasTokenControllerTest.java

```diff
@@ -61,6 +46,16 @@ public void should_throw_exception_when_requested_service_is_not_configured() th
             .isEqualTo("No service configuration found for service nonexistingservice");
     }
 
+    private void assertCanRetrieveSasTokenForService(String serviceName) throws Exception {
+        String tokenResponse = this.mockMvc.perform(get("/token/" + serviceName))
+            .andDo(print())
```

Comment:
no need to print really ;)

Reply:
True. Didn't add this - just extracted common code from two tests.

Reply:
i know. just part of boy scout activities. especially if it's absolutely irrelevant in the first place